tests/unit/challenges/test_views.py

```diff
@@ -115,7 +115,8 @@ def setUp(self):
             'submission_guidelines': 'Submission guidelines for new test challenge',
             'published': False,
             'enable_forum': True,
-            'anonymous_leaderboard': False
+            'anonymous_leaderboard': False,
+            'team_name': 'Test Challenge Host Team'
```

Comment:
@deshraj is putting team name like this wrong? i mean its giving error at line 129 and 124